pkg/utils/doc.go

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package utils
+
+import (
+	"strings"
+
+	"gitlab.com/NebulousLabs/errors"
+)
+
+//CalculateID calculates the ID from an encoded initial document (from create operation)
+func CalculateID(namespace, encodedDocument string, hashAlgorithmAsMultihashCode uint) (string, error) {
```

Comment:
Are these functions generic repository-level utils or can we have more specific packages?

Reply:
Moved one of them to document handler. I will create separate issue to rename utils to docutils.